.ebextensions/django_dev.config

```diff
@@ -8,16 +8,25 @@ option_settings:
     /static: ratenyu/static
 
 container_commands:
-  01_makemigrations:
+  01_create_dir:
+    command: mkdir -p /logs
+  02_change_permissions:
+    command: chmod g+s /logs
+  03_change_owner:
+    command: chown wsgi:wsgi /logs
+  04_makemigrations:
     command: "source /var/app/venv/*/bin/activate && python3 ratenyu/manage.py makemigrations --noinput"
     leader_only: true
-  02_migrate:
+  05_migrate:
     command: "source /var/app/venv/*/bin/activate && python3 ratenyu/manage.py migrate --noinput"
     leader_only: true
-  03_createsu:
+  06_createsu:
     command: "source /var/app/venv/*/bin/activate && python3 ratenyu/manage.py createsu"
     leader_only: true
-  04_collectstatic:
+  07_collectstatic:
     command: "source /var/app/venv/*/bin/activate && python3 ratenyu/manage.py collectstatic --noinput"
-  05_loaddata:
+  08_loaddata:
     command: "source /var/app/venv/*/bin/activate && python3 ratenyu/manage.py load_data"
+  09_createsite:
```

Comment:
Not sure if we want this container command in dev. @ncolosso should this be added?

Reply:
I think we can work on it later and let this merge if everything else is fine?